app/models/post.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
+  belongs_to :user
+  validates :content, presence: { text: true }, length: { maximum: 100 }
```

Comment:
You need to add has_many :posts [odin](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone3/posts2.png)

Reply:
OH yes, thanks, We are goign to check it